### ISSUE TITLE
Customizing toolbar

### ISSUE THREAD

**Issue by PeterC-DLS** [silx-kit/h5web#1377]
### Is your feature request related to a problem?
We have customized some of the toolbar controls to group them together into modal dialogue boxes and move others to the overflow menu.

![colormap-modal](https://user-images.githubusercontent.com/1381719/223065038-54958546-27e1-4e30-8eb1-9374c82cd3db.png)

### Requested solution or feature
To do so, we require some minor changes to existing toolbar components
1. Allow children to be initially added to the overflow menu
2. Export some components
3. ~Allow children to included in a toggle button~

**Comment by PeterC-DLS** [comment]
Thanks, Axel. This is what we have at the moment:
![colormap-tools](https://user-images.githubusercontent.com/1381719/223088531-431f0c3a-9471-4383-a8cb-4e6f6b66b2c7.png)
1. I think a static menu is fine but then in more crowded situations, a 2nd menu may be confusing.
2. Great.
3. Actually, this is not needed any longer as I moved away from re-using toggle button to the vanilla buttion for the `ColorMapOption` button in the screenshot.

**Comment by axelboc** [comment]
1. Makes sense. I could see a pattern where static menus would merge into the overflow menu, as demonstrated here: https://www.telerik.com/kendo-angular-ui/components/toolbar/responsive-toolbar/ — maybe too complicated, I'll have to think about it more. Re-using the overflow menu is definitely easier for the time being.

**Comment by axelboc** [comment]
Closing this as resolved now that `DomainControls` and `DomainSlider` are exported in `main` (as a reminder, the old `DomainSlider` export is now called `DomainWidget`). I won't go further on the idea of having static menus that merge into the overflow menu, at least for the time being. We can open a new issue when the time comes if the idea becomes more relevant.

I probably won't publish the next major release until September, but in the meantime I've published a beta: `v9.0.0-beta.0`. Feel free to try it out and to let me know if the new exports meet your needs. 😉
